abc.tf

```diff
@@ -0,0 +1,11 @@
+resource "azurerm_postgresql_server" "example" {
+  name                         = "example-psqlserver"
+  location                     = "eastus"
+  resource_group_name          = "main_resource_group"
+  administrator_login          = "psqladminun"
+  sku_name                     = "GP_Gen5_4"
+  storage_mb                   = 640000
+  ssl_enforcement_enabled      = false
```

Comment:
:exclamation:**Cycode**: Infrastructure configuration issue: 'Ensure 'Enforce SSL connection' is set to 'ENABLED' for PostgreSQL Database Server'.
Severity: High

Would you like to exclude this IaC violation from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_violation_for_resource_here** | Applies to this resource for this violation in this request only |
| **#cycode_iac_fix_this_violation** | Fix this violation via a commit to this branch |
| **#cycode_iac_false_positive** | Applies to this resource for this violation for all repos in your organization |
| **#cycode_ignore_resource_everywhere** | Applies to this resource for all repos in your organization |
| **#cycode_ignore_violation_for_resource_everywhere** | Applies to this resource for this violation for all repos in your organization |

 <!-- 9b0d095b-1cad-4892-97c5-d3ba22cfdcbd -->

Reply:
#cycode_ignore_violation_for_resource_here

Reply:
:-1: This user is not permitted to perform actions.

Reply:
:-1: This user is not permitted to perform actions.

Reply:
:-1: This user is not permitted to perform actions.

Reply:
:-1: This user is not permitted to perform actions.

Reply:
:-1: This user is not permitted to perform actions.